lib/td/command/query.rb

```diff
@@ -160,6 +160,8 @@ def query(op)
         rescue TreasureData::NotFoundError => e
         end
       end
+    elsif output
+      $stdout.puts "The output file won't be generated. Require `-w` or `--wait` option in addition."
```

Comment:
May I suggest `The output file won't be generated without additional `-w` or `--wait` option` instead?